package/network/utils/xtables-addons/Makefile

```diff
@@ -120,6 +120,23 @@ define Package/iptaccount/install
 endef
 
 
+define Package/iptgeoip
+  $(call Package/xtables-addons)
+  CATEGORY:=Network
+  TITLE:=iptables-mod-geoip support scripts for MaxMind GeoIP databases
+  DEPENDS:=iptables +iptables-mod-geoip \
+		+perl +perlbase-getopt +perlbase-io +perl-text-csv_xs \
+		+wget +gzip +unzip
```

Comment:
Dumb question, but how do you specify a dependency on `wget` (either standalone wget, or Busybox wget, or ...)?

Reply:
> NAK. A package in the LEDE base repo can't depend on a community package.

Well, I could rely on Busybox's wget, unzip, and gunzip and problem solved...

Reply:
LEDE has a uclient-fetch package that disguises as wget and is installed by default.

Reply:
* wget: just depend on uclient-fetch like opkg does
* gzip: is enabled by default in busybox. We generally expect users not to remove busybox applets without replacement, so you can just ignore this dependency
* unzip: you can use a conditional dependency like in https://github.com/freifunk-gluon/packages/blob/6a2d2880064ae580c9c39cd691d4fb76c420412d/admin/autoupdater/Makefile#L13

The remaining issue is perl. Perl is in openwrt/packages, so you can't depend on it from a base package, therefore my suggestion to move the whole thing to openwrt/packages.